### PR TITLE
#106 cant install openssl already installed

### PR DESCRIPTION
**Pull request by tomuben**

### CODE REVIEW

File: db_Test.R [hunk 1]
```diff
@@ -0,0 +1,18 @@
+library(exasol)
+
+drv <- "/home/thomas/Downloads/EXASOL_ODBC-6.2.7/lib/linux/x86_64/libexaodbc-uo2214lv2.so"
```
Comment (morazow) [inline db_Test.R:3]: Is this file intended to be checked in? If so, maybe we can parameterize this path.

**Review by morazow — CHANGES_REQUESTED** [review]
Requested minor change suggestion and question.

File: db_Test.R (hunk 1 above)
Comment (tomuben) [reply db_Test.R:3]: Ops, good catch. I need to remove it.

**Review by morazow — APPROVED** [review]
Looks good!
